src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/LanguageServices/CSharpLanguageFeaturesProvider.cs

```diff
@@ -88,7 +88,7 @@ public string MakeProperNamespace(string name)
         {
             Requires.NotNullOrEmpty(name, nameof(name));
 
-            IEnumerable<string> namespaceNames = name.Split(new char[] { '.' }, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
It's too bad that the `string.Split(char separator, StringSplitOptions options)` overload is only available on CoreCLR